doc/ref_model/chapters/chapter04.md

```diff
@@ -175,129 +174,129 @@ This section covers a list of implicit NFVI capabilities and measurements that d
+| i.pm.007 | Available external storage capacity        | Yes/No      |                                                                                                                                                                                                                             |
 
-<p align="center"><b>Table 4-8:</b> Internal Measurement Capabilities of NFVI</p>
+<p align="center"><b>Table 4-8:</b> Internal Measurement Capabilities of Cloud Infrastructure</p>
 
 <a name="4.1.5"></a>
-### 4.1.5 VIM Capabilities
+### 4.1.5 Cloud Infrastructure management Capabilities
 
-VIM is responsible for controlling and managing the NFVI compute, storage, and network resources. Resources allocation is dynamically set up upon VNFs requirements. This section covers the list of capabilities offered by the VIM to VNFs or service orchestrator.
+Cloud Infrastructure management is responsible for controlling and managing the Cloud Infrastructure compute, storage, and network resources. Resources allocation is dynamically set up upon workloads requirements. This section covers the list of capabilities offered by the VIM to workloads or service orchestrator.
 
 Table 4-9 shows capabilities related to resources allocation
 
 <a name="Table4-12"></a>
 
-| Ref           | VIM Capability                          | Unit   | Definition/Notes                                             |
-|---------------|-----------------------------------------|--------|--------------------------------------------------------------|
-| e.vim.cap.001 | Virtual Compute allocation              | Yes/No | Capability to allocate virtual compute resources  to VNFC    |
-| e.vim.cap.002 | Virtual Storage allocation              | Yes/No | Capability to allocate virtual storage resources  to VNFC    |
-| e.vim.cap.003 | Virtual Networking resources allocation | Yes/No | Capability to allocate virtual networking resources  to VNFC |
-| e.vim.cap.004 | Multi-tenant isolation                  | Yes/No | Capability to isolate resources between tenants              |
-| e.vim.cap.005 | Images management                       | Yes/No | Capability to manage VNFC software images                    |
+| Ref       | Cloud Infrastructure management Capability | Unit   | Definition/Notes                                                 |
+|-----------|--------------------------------------------|--------|------------------------------------------------------------------|
+| e.man.001 | Virtual Compute allocation                 | Yes/No | Capability to allocate virtual compute resources to a workload   |
+| e.man.002 | Virtual Storage allocation                 | Yes/No | Capability to allocate virtual storage resourcesto a workload    |
+| e.man.003 | Virtual Networking resources allocation    | Yes/No | Capability to allocate virtual networking resourcesto a workload |
```

Comment:
Two typos resourcsto didn't make it into the review....

Reply:
Fixed